server/src/main/java/com/cloud/api/ApiSessionListener.java

```diff
@@ -57,17 +52,15 @@ public void sessionCreated(HttpSessionEvent event) {
         synchronized (this) {
             HttpSession session = event.getSession();
             sessions.put(session, event.getSource());
-            sessionCount++;
         }
-        LOGGER.debug("Sessions count: " + sessions);
+        LOGGER.debug("Sessions count: " + sessions.size());
```

Comment:
can we not use getSessionCount() instead?
```suggestion
        LOGGER.debug("Sessions count: " + getSessionCount());
```